lighthouse-core/report/html/report-styles.css

```diff
@@ -541,8 +541,19 @@
   background-color: var(--color-hover);
 }
 
-/* Hide the expandable arrow icon, three ways: via the CSS Counter Styles spec, for webkit/blink browsers, hiding the polyfilled icon */
```

Comment:
link is dead, i found the old sass code but it didn't seem helpful to have here

comment confused me so I just removed. maybe someone could help me reword? we aren't use @counter-styles (whatever that is), and I'm don't know what "polyfilled icon" is referring to.

Reply:
counter styles spec was .... incorrect. 

"polyfilled icon" means the CSS added by deatils-element-polyfill (which is used in PSI for... old edge users and IE11 users?)  
we can remove that polyfill now, tbh.